gems/sorbet-runtime/test/types/props/_props.rb

```diff
@@ -172,7 +172,29 @@ class TestRedactedProps
     prop :secret, String, redaction: [:truncate, 4], sensitivity: []
   end
 
+  describe 'redacted props with no redaction handler' do
+    it 'raises when fetching redacted values' do
+      assert_raises do
```

Comment:
Totally optional: might want to make sure the raised exception has expected class. (If one deleted the `if !handler` in the implementation, I think one would get a `NoMethodError` which is maybe not ideal.)

Reply:
That's a good catch, thanks!